quickjs.c

```diff
@@ -14538,7 +14552,8 @@ static JSValue JS_CallInternal(JSContext *caller_ctx, JSValue func_obj,
     JSInlineCache *ic;
 
 #if defined(DUMP_BYTECODE) && (DUMP_BYTECODE & 128)
-#define DUMP_BYTECODE_OR_DONT(pc) dump_single_byte_code(ctx, pc, b);
+#define DUMP_BYTECODE_OR_DONT(pc) \
+    if (ctx->rt->dump_flags & 128) dump_single_byte_code(ctx, pc, b);
```

Comment:
Why not define this 128 as a flag too?

Reply:
Yes, for completeness, the test should be:
```
#if defined(DUMP_BYTECODE) && (DUMP_BYTECODE & DUMP_BYTECODE_FLAG_STEP) == DUMP_BYTECODE_FLAG_STEP
#define DUMP_BYTECODE_OR_DONT(pc) \
    if (check_dump_flag(ctx->rt, DUMP_BYTECODE_FLAG_STEP)) ...
```

Reply:
More recent version makes all `DUMP_BYTECODE ` variants separate `DUMP_XXX` flags with the same semantics as the other ones.